android/app/src/main/java/com/github/stenzek/duckstation/TouchscreenControllerView.java

```diff
@@ -42,6 +45,8 @@
     private float mMovingLastY = 0.0f;
     private ConstraintLayout mEditLayout = null;
     private int mOpacity = 100;
+    private Map<Integer, View> mFirstHolds = new HashMap<>();
+    private ArrayList<String> mDpadButtons = new ArrayList<String>(Arrays.asList("UpButton", "DownButton", "LeftButton", "RightButton"));
```

Comment:
Probably better to make this an attribute of the button itself, if for some reason we wanted to extend it to non-dpad buttons in the future.

Reply:
That sounds like a great idea actually. Better than what I have here in mind. I guess the buttonview object should have an attribute "is it glidable" during linkButton, as well as a command to fetch it later.